test/llvm/lifetime-intrinsics/lifetime-start-proc.chpl

```diff
@@ -0,0 +1,27 @@
+// This test is used to verify that llvm.lifetime.start instrinsic is being emitted
+// for stack variables of basic type inside a procedure
+
+// CHECK: %[[REG1:[0-9]+]] = bitcast i64* %{{[0-9]+}} to i8*
```

Comment:
It'd be nice to tie this checking to something more specific for the different sections. Could x, y, and z have different integer types, e.g. `int(64)`, `int(32)`, `int(16)`, to do that?